# python-assignment-4

explanation for the tasks 

task 1:
in this task firstly i initilized the cont1 for the first line and then cont 2 for the second line after that i opened the file and read the file in the try part so that if the file not found error occurs for that i then i used except :FileNotFound to handle the error .

task 2:
